openstef/pipeline/optimize_hyperparameters.py

```diff
@@ -137,7 +137,7 @@ def optimize_hyperparameters_pipeline_core(
 
     # Check if sufficient data is left after cleaning
     if not validation.is_data_sufficient(
-        validated_data, pj["completeness_treshold"], pj["minimal_table_length"]
+        validated_data, None, pj["completeness_treshold"], pj["minimal_table_length"]
```

Comment:
Sugestion: Maybe nice to make model an optional argument, this way you do not have to pass None explicitly.